examples/text-replacer/background.js

```diff
@@ -0,0 +1,29 @@
+chrome.commands.onCommand.addListener((command, tab) => {
+  if (command == 'replace-text') {
+    replaceText(tab.id);
+  }
+});
+
+chrome.runtime.onInstalled.addListener(() => {
+  registerContextMenus();
+});
+
+function registerContextMenus() {
+  chrome.contextMenus.create({
+    id: 'replace-text-menuitem',
+    title: 'Replace text',
+  });
+}
+
+chrome.contextMenus.onClicked.addListener((info, tab) => {
+  if (info.menuItemId == 'replace-text-menuitem') {
+    replaceText(tab.id);
+  }
```

Comment:
I'd normally recommend throwing an assert in here, since this is the only id we ever expect.  Similarly for commands.